app/views/follow_requests/_follow_unfollow.html.erb

```diff
@@ -1,17 +1,23 @@
+<div class = "<%= dom_id(sender) %>_<%= dom_id(recipient)%>_follow_unfollow">
```

Comment:
I like the way you added a div. It took me forever to figure out where to add the id!